react-hot-loader/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for react-hot-loader
+// Project: https://github.com/gaearon/react-hot-loader
+// Definitions by: Jacek Jagiello <https://github.com/jacekjagiello/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from "react"
+
+declare module "react-hot-loader" {
+
+  interface ErrorReporterProps {
+    error: any
+  }
+
+  export interface AppContainerProps {
+    children?: React.ReactElement<any>,
+    errorReporter?: React.StatelessComponent<ErrorReporterProps>
```

Comment:
If a change it to `React.ComponentClass`, then stateless component don't work: 

`Type '({error}: ErrorReporterProps) => Element' is not assignable to type 'ReactElement<ErrorReporterProps>'`

But
`React.ComponentClass<ErrorReporterProps> | React.ComponentClass<ErrorReporterProps>`
seems to work. I'am typescript beginner, do you have any better idea?